templates/Angular2Spa/ClientApp/app/app.module.client.ts

```diff
@@ -0,0 +1,21 @@
+import { NgModule } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+import { FormsModule } from '@angular/forms';
+import { HttpModule } from '@angular/http';
+import { sharedConfig } from './app.module';
```

Comment:
Sorry just seeing this now!

Did you have boot-client & boot-server using browser-app.module & server-app.module ?

To get rid of the ngIf unknown errors, you just import CommonModule into your base app.module NgModule (https://github.com/MarkPieszak/aspnetcore-angular2-universal/blob/master/Client/app/app.module.ts#L51)

You can see in there I have the app.module importing the Routes, and all the base declarations etc that can be shared.  Browser & Server-app.module's both import the app.module and they each declare the `bootstrap: [ AppComponent ]` themselves. You want to make sure app.module doesn't declare a bootstrap Component.